doc/memos/rdm-draft-alamos-ieee802154-radio-hal.md

```diff
@@ -635,7 +630,7 @@ MIB.
     int (*get_tx_status)(ieee802154_dev_t *dev, ieee802154_tx_info_t *info);
 
     /**
-     * @brief Set IEEE802.15.4 addresses in hardware address filter
+     * @brief Set IEEE802.15.4 address in hardware address filter
```

Comment:
Hmmm in theory there are 2 addresses and the PAN ID (which can be also considered as an address). Should it still be "address"?

Reply:
Anyway, I will merge this file and adapt later as needed :). Thanks!